pkg/scheduler/metrics/metrics.go

```diff
@@ -195,11 +195,14 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
-	PreemptionVictims = metrics.NewGauge(
-		&metrics.GaugeOpts{
-			Subsystem:      SchedulerSubsystem,
-			Name:           "pod_preemption_victims",
-			Help:           "Number of selected preemption victims",
+	PreemptionVictims = metrics.NewHistogram(
+		&metrics.HistogramOpts{
+			Subsystem: SchedulerSubsystem,
+			Name:      "pod_preemption_victims",
+			Help:      "Number of selected preemption victims",
+			// we think #victims>50 is pretty rare, therefore [50, +Inf(100)) is considered a single bucket.
+			// reminder: max number of pods per node is 100
```

Comment:
Sorry to hav given inaccurate suggestions. I think the # pods per node is configurable and not necessarily 100 although some cloud providers have a specific limit, e.g., GKE has 110.
I don't know if there is a written limit in kubernetes, this issue #23349 seems to suggest that the current limit is 110.

To be safe, remove the "100s" from the comment.

Reply:
I removed the "100s".